index.html

```diff
@@ -229,28 +231,25 @@ <h4>Skills</h4><img class="arrow" src="images/right.png" alt="">
     <!-- contact form -->
     <section class="contact_form">
         <div class="contact_wrapper">
-            <h1>Contact me</h1>
-            <p>
-                If you have an application you are interested in developing,
-                a feature that you need built or a project that needs coding.
-                I’d love to help with it
-            </p>
-            <form action="https://formspree.io/f/xbjeloeq" method="post">
-                <ul>
-                    <li>
-                      <input name="username" type="text" placeholder="Your name" maxlength="30"required>
-                    </li>
-                    <li>
-                      <input name="email" type="email" placeholder="youremail@example.com" required>
-                    </li>
-                    <li>
-                      <textarea name="message" id="message" placeholder="Write your message here" maxlength="500" required></textarea>
-                    </li>
-                </ul>
-                <input type="submit" value="Get It Touch">
-            </form>
+            <div class="form_wrapper">
+                <h1>Contact me</h1>
+                <p>
+                    If you have an application you are interested in developing,
+                    a feature that you need built or a project that needs coding.
+                    I’d love to help with it
+                </p>
```

Comment:
- [ ] Kindly decrease the width of this paragraph as per the design template, and just like that perfection is around the corner. 💯️

**Your Design**

![Screenshot from 2023-01-19 17-53-22](https://user-images.githubusercontent.com/17243325/213491488-9fdc837c-f961-4a27-bd97-54696e4331f2.png)

**Figma Design**

![Screenshot from 2023-01-19 17-53-42](https://user-images.githubusercontent.com/17243325/213491591-3e2a05b2-4960-46c4-a7d1-dfe185406d98.png)